docs/ObjectSearchCreate.md

```diff
@@ -0,0 +1,38 @@
+## Features
+
+```ts
+interface IFieldConfig {
+  // [...]
+  populateFromSearch: boolean;
+  populateNameFromSearch: boolean;
+  // [...]
+}
+
+
+export interface IFieldConfigObjectSearchCreate extends IFieldConfigBase {
+  createFields: IFieldConfigBase[];
+  endpoint: string;
+  searchFilters?: { [key: string]: any };
+}
+```
+
+A FieldConfig of type `objectSearchCreate` defines:
+
+- `endpoint` which it will submit it's `?search=` queries to
+- `createFields` a FieldSet to allow the user to compose their own
+  object if they are unable to find one that matches their query
+- `searchFilters` (optional) params to include alongside the search
+  
+The create functionality is currently blocked until the user types  at least
+3 characters, but this will be customizable in the future.
+
+Create fields may include these optional attributes:
+
+- `populateFromSearch` to default the value of this field to the user's search 
+  string
+- `populateNameFromSearch` to default the value of a field ending in
```

Comment:
This is a little unclear; it doesn't quite get at the idea that there can be both a `first_name` and a `last_name` marked with `populateNameFromSearch`. Though I guess with the example it indicates that